### ISSUE TITLE
Auto remove for slashes ( unslashes )

### ISSUE THREAD

**Issue by hrizal** [josdejong/jsoneditor#1245]
maybe it is for next feature

**Comment by josdejong** [comment]
Rizal do you mean slashes like `\n`? Those are necessary escape characters needed for example for return, tab, etc. In that case I think this is a duplicate of #185.

**Comment by hrizal** [comment]
usually if we copy and paste from the JSON format in the form of a JSON string

example:
```
{
  "name": "rizal",
  "address": "{\" home \ ": \" road no 1 \ ", \" office \ ": \" road no 2 \ "}"
}

```
and we try to copy the contents of the address

`{\ "home \": \ "road no 1 \", \ "office \": \ "road no 2 \"}`

![image](https://user-images.githubusercontent.com/47385704/108800485-d8b9e280-75c5-11eb-8202-b98b937c4b7c.png)

**Comment by josdejong** [comment]
Ah, now I understand you. I've encountered those cases too. Would be very nice if we could repair that. I've opened an issue for it in the `jsonrepair` library: https://github.com/josdejong/jsonrepair/issues/16

**Comment by josdejong** [comment]
Repairing of escaped strings containing JSON is now available (`jsoneditor@9.3.0`, web application `v5.7.0`)
